docker/python2/onbuild/docker-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+set -o errexit
+
+# Execute whatever command was given
+exec "$@"
```

Comment:
If we do need this entrypoint file we need to only have it exist in one place, we can't have the same file in 4 places.